exporters/webhook/app.py

```diff
@@ -0,0 +1,202 @@
+#!/usr/bin/env python3
+#
+# Copyright Red Hat
+#
+#    Licensed under the Apache License, Version 2.0 (the "License"); you may
+#    not use this file except in compliance with the License. You may obtain
+#    a copy of the License at
+#
+#         http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#    WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#    License for the specific language governing permissions and limitations
+#    under the License.
+#
+
+import asyncio
+import http
+import json
+import logging
+
+from fastapi import Depends, FastAPI, Header, Request, Response
+from fastapi.responses import PlainTextResponse
+from prometheus_client import Counter, generate_latest
+from prometheus_client.core import REGISTRY
+from pydantic import BaseModel, ValidationError, parse_obj_as
+
+import pelorus
+from committime import CommitMetric
+from pelorus.config import load_and_log
+from webhook.models.github_webhook import GitHubCommit, GitHubDeliveryHeaders
+from webhook.models.in_memory_metric import (
+    PelorusGaugeMetricFamily,
+    in_memory_commit_metrics,
+)
+from webhook.models.pelorus_webhook import (
+    CommitTimePelorusPayload,
+    PelorusDeliveryHeaders,
+)
+
+pelorus.setup_logging()
+
+webhook_received = Counter("webhook_received_total", "Number of received webhooks")
+webhook_processed = Counter("webhook_processed_total", "Number of processed webhooks")
+
+commit_hash_set = set()
+
+# APIRouter here... ?
```

Comment:
you can, I prefer to put the routes in a separate folder, and then you need the router

Reply:
ok will leave as it is for now as it's poc.